pkg/webhook/mutation.go

```diff
@@ -35,11 +35,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-type mutationResponse string
-
 const (
-	mutationSkipResponse    mutationResponse = "skip"
-	mutationUnknownResponse mutationResponse = "unknown"
+	mutationSkipResponse    requestResponse = "skip"
+	mutationUnknownResponse requestResponse = "unknown"
```

Comment:
Will we also need `error` and `success`?

Reply:
Done